### PR TITLE
PTI-257: Delete functionality for documents.

### PR DESCRIPTION
**Pull request by danieltruong**

### CODE REVIEW

File: angular/projects/admin-nrpti/src/app/records/orders/order-add-edit/order-add-edit.component.html [hunk 1]
```diff
@@ -161,38 +150,18 @@ <h4>Outcome Details</h4>
...
-            class="form-control"
-          ></textarea>
+          <textarea name="outcomeDescription" id="outcomeDescription" type="text" formControlName="outcomeDescription"
+            class="form-control"></textarea>
         </div>
       </div>
     </section>
 
-    <section>
+    <section *ngIf="currentRecord?.documents">
```
Comment (NickPhura) [inline angular/projects/admin-nrpti/src/app/records/orders/order-add-edit/order-add-edit.component.html]: Is `?.` supported?

I thought it wasn't released yet?

Comment (danieltruong) [reply angular/projects/admin-nrpti/src/app/records/orders/order-add-edit/order-add-edit.component.html]: It should be supported.

Comment (marklise) [reply angular/projects/admin-nrpti/src/app/records/orders/order-add-edit/order-add-edit.component.html]: It's only supported in tsc 3.7, our package.json is referencing ~3.2

Comment (NickPhura) [reply angular/projects/admin-nrpti/src/app/records/orders/order-add-edit/order-add-edit.component.html]: Looks like its only supported in a select few browser versions: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Probably best to just use `&&` in the meantime

File: angular/projects/admin-nrpti/src/app/records/orders/order-add-edit/order-add-edit.component.html [hunk 2]
```diff
@@ -44,14 +44,8 @@ <h4>Basic Information</h4>
...
-            <input
-              class="form-control"
-              placeholder="yyyy-mm-dd"
-              name="dp"
-              formControlName="dateIssued"
-              ngbDatepicker
-              #d="ngbDatepicker"
-            />
+            <input class="form-control" placeholder="yyyy-mm-dd" name="dp" formControlName="dateIssued" ngbDatepicker
+              #d="ngbDatepicker" />
```
Comment (marklise) [inline angular/projects/admin-nrpti/src/app/records/orders/order-add-edit/order-add-edit.component.html]: Keep the formatting vertical

**Pull request merged**